### PR TITLE
chore: update marked to version 0.6.0

### PR DESCRIPTION
**Pull request by lpinca**
The new release includes https://github.com/markedjs/marked/pull/1383.
Would it be possible to also release a new version of redoc and redoc-cli?

Thank you.

### CODE REVIEW

**Comment by lpinca** [comment]
Opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31834 to address the failures, will update accordingly here when possible.

**Comment by ChrisDillinger** [comment]
Hi there! Can this PR be merged? What else needs to be reviewed so that this PR can be successfully closed out? I am very much looking forward to this change here: https://github.com/Rebilly/ReDoc/commit/809e7f2841b4cb034e8d23e92a3ebfef23580022

As I really want to integrate this wonderful open source tool into my project. Thanks so much guys for being such a cool community.

File: src/services/MarkdownRenderer.ts [hunk 1]
```diff
@@ -129,7 +131,7 @@ export class MarkdownRenderer {
         this.currentTopHeading && this.currentTopHeading.id,
       );
     }
-    return this.originalHeadingRule(text, level, raw);
+    return this.originalHeadingRule(text, level, raw, this.slugger);
```
Comment (RomanGotsiy) [inline src/services/MarkdownRenderer.ts]: Looks like you don't need to manually create `slugger`.
Just pass it through function parameters:

```ts
headingRule = (text: string, level: number, raw: string, slugger: marked.Slugger) => {
  // ...
  return this.originalHeadingRule(text, level, raw, slugger);
}
```

Comment (lpinca) [reply src/services/MarkdownRenderer.ts]: Good catch, addressed.

**Comment by RomanGotsiy** [comment]
Thanks!

**Pull request merged**